plugins/woocommerce/tests/legacy/unit-tests/woocommerce-admin/api/reports-taxes.php

```diff
@@ -134,7 +157,7 @@ public function test_get_reports_taxes_param() {
 				'tax_rate_id'       => 1,
 				'tax_rate'          => '7',
 				'tax_rate_country'  => 'US',
-				'tax_rate_state'    => 'GA',
+				'tax_rate_state'    => 'NY',
```

Comment:
Any reason for this change? I mean, I'm partial to NY 😄, but there doesn't seem to be a need for this change.